src/app/backing/components/AllocationBar/AllocationBarTooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import { motion, AnimatePresence } from 'motion/react'
+import { useState } from 'react'
+import { b } from 'vitest/dist/chunks/suite.d.FvehnV49.js'
+
+interface AllocationBarTooltipProps {
+  builderAddress: string
+  currentBacking: number
+}
+
+export const AllocationBarTooltip = ({ builderAddress, currentBacking }: AllocationBarTooltipProps) => {
+  console.log(builderAddress)
+
+  return (
+    <motion.div
+      initial={{ opacity: 0, y: -5, scale: 0.95 }}
+      animate={{ opacity: 1, y: 0, scale: 1 }}
+      exit={{ opacity: 0, y: -5, scale: 0.95 }}
+      transition={{ duration: 0.2, ease: 'easeOut' }}
+      className="flex-col p-2 hidden group-hover:flex absolute -top-8 left-5 bg-st-white rounded-md border z-[100000]"
```

Comment:
the background is different from the one we've in the design. I'd suggest that we use the Tooltip component used in other places.